src/test/resources/dataset-bags/additional-license-by-text/metadata/dataset.xml

```diff
@@ -9,6 +9,7 @@
         <dc:description>A dataset with some files but no additional license</dc:description>
         <dc:creator>A Creator</dc:creator>
         <ddm:created>2015</ddm:created>
+        <ddm:available>2016</ddm:available>
```

Comment:
I think `ddm:available` and `ddm:created` should be formatted according to the ISO standard. So this isn't correct, I guess...